test/compress/issue-1001.js

```diff
@@ -0,0 +1,8 @@
+parenthesis_strings_in_parenthesis: {
+    input: {
+        ('(');
+        a(')');
+
+    }
+    expect_exact: '"(";a(")");'
```

Comment:
:worried: I don't like to make those kinds of assumptions. "good enough" is rarely ever good enough.